walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/MeetingRequest.java

```diff
@@ -40,8 +40,13 @@
   private final long duration;
 
   public MeetingRequest(Collection<String> attendees, long duration) {
+    this(attendees, Collections.EMPTY_LIST ,  duration);
```

Comment:
nit: remove whitespace before `,` and extra whitespace (there are two) after it.